openvpn/osfamilymap.yaml

```diff
@@ -5,6 +5,10 @@ Arch:
 Debian:
   group: nogroup
   log_user: root
+  client_conf_dir: /etc/openvpn/client
+  client_service: openvpn-client
+  server_conf_dir: /etc/openvpn/server
+  server_service: openvpn-server
```

Comment:
I should have put these new config values under a `client` and `server` dict?, something like this
```
client:
  conf_dir: /etc/openvpn/client
  service: openvpn-client
server:
  conf_dir: /etc/openvpn/server
  service: openvpn-server
```

Reply:
:+1: